lib/model/country_code.dart

```diff
@@ -0,0 +1,9 @@
+const ANDORRA = 'AD';
```

Comment:
Shouldn't `ANDORRA` be "`ad`" instead of "`AD`"?

Reply:
Yes! Thank you for being alert.